src/auth/AuthRoute.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Navigate } from 'react-router-dom';
+import { useMagicAuth } from './useMagicAuth';
+
+export interface IAuthRouteProps {
```

Comment:
It's generally advised to avoid using prefixes like "I". It's better to use specific names or skip interfaces altogether. Just my two cents on the matter.